<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dima Fedorov Levit</author>
 <class>ControlsForm</class>
 <widget class="QMainWindow" name="ControlsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BioView Remote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="1">
     <widget class="QGroupBox" name="fileGroup">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>File</string>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="fileEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseBtn">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="6">
     <widget class="DSubAreaLabel" name="thumbLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="scaleGroup">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Scale</string>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="zoomOutBtn">
         <property name="text">
          <string>(-)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zoom11Btn">
         <property name="text">
          <string>1:1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zoomInBtn">
         <property name="text">
          <string>(+)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QGroupBox" name="metaGroup">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Metadata</string>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTextEdit" name="metaEdit">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="pagesGroup">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Pages</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="2">
        <widget class="QPushButton" name="pagePlusBtn">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pageMinsBtn">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pageZeroBtn">
         <property name="text">
          <string>Initial</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="enhanceGroup">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Live enhancement</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="enhanceLabel">
         <property name="text">
          <string>Enhancement:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="enhanceCombo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="chanGroup">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Channels</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QWidget" name="rgbChannelsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="2" column="2">
           <widget class="QComboBox" name="blueCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="blueLabel">
            <property name="text">
             <string>Blue:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="redCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Red channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="greenLabel">
            <property name="text">
             <string>Green:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="greenCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Green channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="redLabel">
            <property name="text">
             <string>Red:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="blueCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Blue channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="greenCombo"/>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="redCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="moreChannelsLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>&lt;a href=&quot;#&quot;&gt;more...&lt;/a&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="exChannelsWidget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="yellowCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Red channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="yellowCombo">
            <property name="minimumSize">
             <size>
              <width>97</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="magentaLabel">
            <property name="text">
             <string>Magenta:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="magentaCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Green channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" rowspan="2">
           <widget class="QComboBox" name="magentaCombo">
            <property name="minimumSize">
             <size>
              <width>97</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1" rowspan="2">
           <widget class="QCheckBox" name="cyanCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Trigger visibility of the Blue channel</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="cyanLabel">
            <property name="text">
             <string>Cyan:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="cyanCombo">
            <property name="minimumSize">
             <size>
              <width>97</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="yellowLabel">
            <property name="text">
             <string>Yellow:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWall">
    <property name="title">
     <string>Wall</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionFullVirtualScreen"/>
    <addaction name="actionShowMetaData"/>
    <addaction name="actionShowScaleBar"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure_wall_host"/>
    <addaction name="separator"/>
    <addaction name="actionCloseWall"/>
   </widget>
   <widget class="QMenu" name="menuModify">
    <property name="title">
     <string>Modify</string>
    </property>
    <addaction name="actionRotate_Right"/>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_180_deg"/>
    <addaction name="separator"/>
    <addaction name="actionProject_all_planes_by_maximum_intesity"/>
    <addaction name="actionProject_all_planes_by_minimum_intesity"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionUse_Smooth_Zoom"/>
    <addaction name="actionUse_Dynamic_Enhancement"/>
   </widget>
   <addaction name="menuWall"/>
   <addaction name="menuView"/>
   <addaction name="menuModify"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFullVirtualScreen">
   <property name="text">
    <string>Full Virtual Screen</string>
   </property>
  </action>
  <action name="actionCloseWall">
   <property name="text">
    <string>Close wall</string>
   </property>
  </action>
  <action name="actionShowMetaData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show meta-data</string>
   </property>
  </action>
  <action name="actionShowScaleBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show scale bar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="text">
    <string>Rotate Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="text">
    <string>Rotate Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRotate_180_deg">
   <property name="text">
    <string>Rotate 180 deg</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUse_Smooth_Zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Smooth Zoom, Note: Slow on WALL!</string>
   </property>
  </action>
  <action name="actionUse_Dynamic_Enhancement">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Dynamic Enhancement (Only Visible Area), Note: Slow on WALL!</string>
   </property>
  </action>
  <action name="actionProject_all_planes_by_maximum_intesity">
   <property name="text">
    <string>Project all planes by maximum intesity</string>
   </property>
  </action>
  <action name="actionProject_all_planes_by_minimum_intesity">
   <property name="text">
    <string>Project all planes by minimum intesity</string>
   </property>
  </action>
  <action name="actionConfigure_wall_host">
   <property name="text">
    <string>Configure wall host</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="2" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>DSubAreaLabel</class>
   <extends>QLabel</extends>
   <header>subAreaLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
